groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -117,6 +116,7 @@ BSLS_IDENT("$Id: $")
 #include <bsls_util.h>
 
 #include <stddef.h>
+#include "bslstl_inplace.h"
```

Comment:
Use `<>` for include, put this include right after `bslstl_badoptionalaccess.h`  

Note that includes from same package come first, and includes should within the same group should be in alphabetic order - bde-format can do the sorting.

Reply:
That's the second time you've stumbled upon ""  include which is weird because I would have just copy pasted a previous line and modified the header name (I'm a copy paste addict). I wonder if this is some Eclipse trickery. Will pay attention in the future. fixed.